Module-15-Assignment/LWC/createContact.js

```diff
@@ -0,0 +1,30 @@
+import { LightningElement, api, track } from 'lwc';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+
+export default class CreateContact extends LightningElement {
+
+    @api recordId;
+    @track contactId;
+    handleSuccess(event) {
+        this.contactId = event.detail.id;
+        this.recordId = this.contactId;
```

Comment:
redirect to record detail page

Reply:
Navigation is currently not supported in lwc